packages/block-editor/src/components/block-card/index.js

```diff
@@ -43,7 +43,10 @@ function BlockCard( { title, icon, description, blockType, className } ) {
 	}, [] );
 
 	const { selectBlock } = useDispatch( blockEditorStore );
-
+	// As with Navigation block it's not ideal having Pattern block specific code here, but only synced patterns are selectable in the editor
+	// currently and finding the sync status higher up involves a call to the core-data store which isn't allowed in the block editor.
+	const blockTitle =
+		title === __( 'Pattern' ) ? __( 'Synced Pattern' ) : title;
```

Comment:
Is updating the `useBlockDisplayInformation` hook an option so the `BlockCard` just gets passed the appropriate title?

We already check there if the current block type is a reusable block or template part, so it might not be a stretch to assign the pattern's name to the title property there as well.

Reply:
Yeh, that is what I was planning to do, will try and get back to it today.